packages/SwingSet/src/controller/initializeSwingset.js

```diff
@@ -469,11 +498,11 @@ export async function initializeSwingset(
   const kconfig = {
     ...config,
     namedBundleIDs: {},
-    idToBundle: {
+    idToBundle: sortObjectProperties({
       ...idToNamedBundle,
       ...idToVatBundle,
       ...idToDeviceBundle,
-    },
+    }),
```

Comment:
For a second I thought we'd need to sort the `Object.keys(nameToBundle)` on line 507 below, because `processGroup` takes `config.vats`/etc as inputs. But then I realized that the `sort` of `config.vats`/etc all happens well before this, so the results will still be deterministic.

It might still be nice to sort those, so the DB writes appear in a sorted order, but it's strictly cosmetic.